src/Microsoft.AspNetCore.Mvc.Localization/project.json

```diff
@@ -1,5 +1,10 @@
 {
-  "description": "Features that enable globalization & localization of MVC applications.",
+  "description": "ASP.NET Core MVC features that enable globalization and localization of applications.\r\nCommonly used types:\r\nMicrosoft.AspNetCore.Mvc.Localization.IHtmlLocalizer\r\nMicrosoft.AspNetCore.Mvc.Localization.IHtmlLocalizer<TResource>",
```

Comment:
`IHtmlLocalizer` will appear much less than `IHtmlLocalizer<TResource> in user code. The base is primarily useful for our extension methods et cetera.

I suggest
```
Microsoft.AspNetCore.Mvc.Localization.IHtmlLocalizer<TResource>
Microsoft.AspNetCore.Mvc.Localization.IViewLocalizer
```